src/cmd/logic-function.test.js

```diff
@@ -340,7 +338,7 @@ describe('LogicFunctionCommands', () => {
 
 			const paths = ['dir/', 'dir/path/to/file'];
```

Comment:
Might be good to change this since paths[0] is not being used anymore, maybe you just need to define a `jsonPath` field and pass it to your function (will be more clean and clear)